internal/cli/apis.go

```diff
@@ -55,6 +57,12 @@ var (
 		Help:         "Whether Refresh Tokens can be issued for this API (true) or not (false).",
 		AlwaysPrompt: true,
 	}
+	apiSigningAlgorithm = Flag{
+		Name:      "Signing Algorithm",
+		LongForm:  "signing-alg",
+		ShortForm: "a",
```

Comment:
Can we remove the short arg? `a` isn't immediately clear about what it associates with. Further, users could get confused with `-s`, is it signing alg or scopes?